tasks/HandMeThat.tex

```diff
@@ -1,91 +1,61 @@
-\section{Hand Me That [Party Host]}
+\section{Hand Me That}
 \label{test:hand-me-that}
-A guest at the party speaks English, but with only a limited vocabulary. The robot will assist them in obtaining things that they gesture for.
+A guest asks the robot to identify objects in the household.
```

Comment:
This is a confusing (and disappointing?) test for the audience, as they expect the robot to actually bring the pointed-at object. 
Maybe it already helps for the robot to ask if you want that object and for the operator to only confirm this for the 5th object. Then, getting to the 5th object and awarding points for the 5th object to be grasped is an incentive to get to 5 objects fast.

Reply:
Yes, I already started a discussion in the TC group about changing this test to a robot does inventory test. Ca: Operator shows coke, robot needs to find out how many coke there are in the kitchen, bonus if it finds out how many drinks in general there are. Question is, are teams still fine with a heavy rework of a test at this point in time?

Reply:
Nope, I would not rework a test in such a major way for such little reward at this point.

Reply:
Yeah fair enough. I think I still want to change the name. Currently its not really describing the test. That would probably also make it less confusing for audiences.

Reply:
Changed name